deploy/helm/flightctl/templates/_db-migration-job.tpl

```diff
@@ -0,0 +1,192 @@
+{{/*
+Renders the DB migration Job. Supports dry-run variant and hook customization.
+Parameters:
+- context: Root context (.)
+- name: Job name
+- hooks: List of hook phases (e.g., (list "pre-upgrade"))
+- hookWeight: String weight for execution order
+- hookDeletePolicy: Hook delete policy
+- isDryRun: If true, runs migration with --dry-run and skips side-effect steps
+*/}}
+{{- define "flightctl.dbMigrationJob" }}
+{{- $ctx := .context }}
+{{- $name := .name }}
+{{- $hooks := .hooks | default (list) }}
+{{- $hookWeight := .hookWeight | default "10" }}
+{{- $deletePolicy := .hookDeletePolicy | default "hook-succeeded" }}
+{{- $isDryRun := .isDryRun | default false }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ $name }}
+  namespace: {{ default $ctx.Release.Namespace $ctx.Values.global.internalNamespace }}
+  labels:
+    app: flightctl-db-migration
+    release: {{ $ctx.Release.Name }}
+  annotations:
+    {{- if gt (len $hooks) 0 }}
+    helm.sh/hook: {{ join "," $hooks }}
+    {{- end }}
+    helm.sh/hook-weight: "{{ $hookWeight }}"
+    helm.sh/hook-delete-policy: "{{ $deletePolicy }}"
+spec:
+  backoffLimit: {{ $ctx.Values.dbSetup.migration.backoffLimit | default 3 | int }}
+  activeDeadlineSeconds: {{ $ctx.Values.dbSetup.migration.activeDeadlineSeconds | default 600 | int }}
+  template:
+    metadata:
+      labels:
+        app: flightctl-db-migration
+        release: {{ $ctx.Release.Name }}
+    spec:
+      restartPolicy: Never
+      serviceAccountName: flightctl-db-migration
+      initContainers:
+      {{- include "flightctl.databaseWaitInitContainer" (dict "context" $ctx "userType" "admin" "timeout" 120 "sleep" 2) | nindent 6 }}
+      {{- if not $isDryRun }}
+      - name: setup-database-users
+        image: "{{ $ctx.Values.dbSetup.image.image }}:{{ $ctx.Values.dbSetup.image.tag | default $ctx.Chart.AppVersion }}"
+        imagePullPolicy: {{ default $ctx.Values.global.imagePullPolicy $ctx.Values.dbSetup.image.pullPolicy }}
+        env:
+        - name: DB_HOST
+          value: "{{ include "flightctl.dbHostname" $ctx }}"
+        - name: DB_PORT
+          value: "{{ (default 5432 $ctx.Values.db.port) }}"
+        - name: DB_NAME
+          value: "{{ (default "flightctl" $ctx.Values.db.name) }}"
```

Comment:
these `default` values are the defaults from values.yaml. Why specify them again ?
```suggestion
        - name: DB_PORT
          value: "{{ $ctx.Values.db.port }}"
        - name: DB_NAME
          value: "{{ $ctx.Values.db.name }}"
```